randovania/__init__.py

```diff
@@ -172,7 +172,9 @@ def setup_logging(default_level: str, log_to_file: Path | None, quiet: bool = Fa
 if is_dirty():
     _final_version += "-dirty"
 
+UNKNOWN_VERSION = "0.0.0"
+UNKNOWN_GIT_HASH = b"UNKN"
 
 __version__ = _final_version
-VERSION = _final_version
-GIT_HASH = version_hash.git_hash
+VERSION: str = _final_version if _final_version is not None else UNKNOWN_VERSION
+GIT_HASH: bytes = version_hash.git_hash if version_hash.git_hash is not None else UNKNOWN_GIT_HASH
```

Comment:
I believe these can just be `version_hash.git_hash or UNKNOWN_GIT_HASH`? unless you anticipate a situation in which `b""` is a valid hash

Reply:
Sure. Usually prefer to be more explicit, but I don't think we can even ever get `b""` for git_hash here.

Reply:
`b""` is an invalid value for git_hash.

Reply:
we can do the same for the version too, right? I'm pretty sure `""` is an invalid value there as well

Reply:
It is, the version needs to b e `X.Y.Z`